content/docs/user-guide/dropbox-privacy.md

```diff
@@ -0,0 +1,78 @@
+# Privacy Policy for Access to Dropbox API
+
+DVC is using the [Dropbox API](https://www.dropbox.com/developers) to make it
+easier to store files on Dropbox. For more details on how to set it up, refer to
+[Setup a Dropbox DVC Remote](/doc/user-guide/setup-dropbox-remote).
+
+DVC uses the _DVC_ Application on Dropbox. (That is the name you will see in a
+consent screen.) This integration is governed by common policies, recorded here.
+
+Your use of Dropbox API with DVC is subject to each API’s respective terms of
+service. See [Dropbox Terms of Service](https://www.dropbox.com/terms) and
+[Dropbox Privacy Regulations](https://www.dropbox.com/privacy).
+
+## Accessing user data
+
+DVC accesses Dropbox resources from the machine where you use DVC. Your machine
+communicates directly with the Dropbox API.
+
+The _DVC_ Application never receives your data or the permission to access your
+data. The owners of the project can only see anonymous, aggregated information
+about usage of tokens obtained through its OAuth client, such as which APIs and
+endpoints are being used.
+
+DVC sends users to `https://www.dropbox.com/oauth2/authorize` for authorization
+and does not directly access or collect user data used by Dropbox Auth.
+
+## Using user data
+
+DVC includes functions that you can execute in order to read or modify your own
+data (or data shared with you). This can only happen after you provide a token,
+which requires that you authenticate yourself as a specific Dropbox user and
+authorize these actions.
+
+DVC can help you get a token by guiding you through the OAuth flow in the
+browser. There you must consent to allow the _DVC_ Application to operate on
+your behalf. The OAuth consent screen will describe the scope of what is being
+authorized e.g. it will name the target API(s) and whether you are authorizing
+“read only” or “read and write” access.
+
+## Scopes
+
+DVC uses the `Scoped App (App Folder)` permission type. Therefore its access is
+limited to the `DROPBOX_HOME/Applications/dvc` directory. There, the following
```

Comment:
Can it be shared though? Or is it unique per user?

The main use case for DVC is the shared remote ...

Reply:
Also Dropbox has Teams, not sure how Application scope applies to them

Reply:
The subdirectories could be shared, e.g. like one per project, so one would be sharing a specific project only. Not sure how it would work with Teams, I do not have such subscription.

Reply:
> The subdirectories could be shared, e.g. like one per project, so one would be sharing a specific project only. 

not sure I follow, could you clarify a bit? Let's say my dropbox account is `ivan@dvc.org` and yours it `
Grzegorz @ Mrukwa.com` . Now, I do `dvc remote add -d dropbox://my-project`. It will create `Applications/dvc/my-project` under my Dropbox account. Can I share `Applications/dvc/my-project` with you, so that you have access to it? Can you put into a different path `DROPBOX_HOME/path` (which means that `dvc` app won't have access to it, right)?

(not saying that we are doing it wrong, trying to clarify these nuances)

Reply:
I assumed that one is capable of sharing the `Applications/dvc/my-project` directory and the other person is able to place it under `Applications/dvc/my-project` by themselves. The default location of incoming shares is the root directory of Dropbox.

But, as you indicated [above](https://github.com/iterative/dvc.org/pull/1896#discussion_r521844937), it's not so straightforward, so we probably need to change the Dropbox App scope for DVC.